src/shady/generator/CMakeLists.txt

```diff
@@ -1,7 +1,17 @@
 find_package(json-c REQUIRED)
 
+if(WIN32)
```

Comment:
this is fine but if you want you can make it a global toggle instead to deal with all dependencies statically (where applicable)